### ISSUE TITLE
How to run the test

### ISSUE THREAD

**Issue by Thorn323** [ableplayer/ableplayer#169]
I have donwload the whole package and tried to run the tests, e.g. ableplayer-ableplayer-f43ef8d/tests/test6.html. However, they are not working. It seems that the scripts are not loading. Any idea of what is happening?

**Comment by sebastiankelle** [comment]
I just tested http://ableplayer.github.io/ableplayer/tests/test6.html and it seems to work just fine. There has been no recent change on the ableplayer main branch, only the development change is currently being modified. Even so, also there all tests should work. Did you check for dependencies?

**Comment by Thorn323** [comment]
Yes, the problem occurs just donwloading the files into my desktop. Opening the code on the browser after loading the test (Ctrl+U in Firefox), and clicking on the dependency links there is no effect with
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
I have tried substutiting by src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js" but it seems useless.

Thank you for the quick answer

**Comment by terrill** [comment]
@Thorn323 try downloading from the latest version from the [develop branch] (https://github.com/ableplayer/ableplayer/tree/develop). We've fixed some bugs that might be affecting your instance.

**Comment by Thorn323** [comment]
Unfortunately, I am experiencing the same problem with that version.

**Comment by terrill** [comment]
If you're just running it as a local file (i.e., not on a web server) then there may indeed be problems. As you've noted, the test files load jquery via CDN using protocol-independent URLs (e.g., "//" rather than "https://"). If the URL of the test page is a local file ("file://") then the URL won't work and jquery won't load. I would expect changing the jquery script src to "https://" to correct the query problem, but there may be other problems. For example, Firefox may have cross-domain font restrictions in place that prevent icon fonts from loading properly. 

The bottom line is: Able Player was really designed to run on the web, rather than as a local player. If you need to run it locally then I would recommend setting up a local web server using a tool like [XAMPP](https://www.apachefriends.org/index.html) or [MAMP](https://www.mamp.info/en/).

**Comment by Thorn323** [comment]
That was exactly the problem. Thank you for your help and your great job.
